dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -130,7 +130,7 @@ class SentryStackTraceFactory {
       }
     }
 
-    if (frame.isCore || frame.uri.scheme == 'package') return false;
+    if (frame.isCore) return false;
```

Comment:
@marandaneto we can simply remove this condition, but I'm not sure that we still need the `inAppIncludes` loop WDYT ?

Reply:
so `frame.isCore` means:
  /// Whether this stack frame comes from the Dart core libraries.

is there a way to also check for `frame.isFlutterCore`?

if we remove `frame.uri.scheme == 'package'`, I still want the `Flutter` core packages to be `inApp=false`, if we solve that, it'd be a win already

Reply:
what about this ?

```dart
if (frame.isCore ||
        (frame.uri.scheme == 'package' && frame.package == 'flutter'))
      return false;
```

Reply:
looks good, could you try on the samples and see if they are marked `inApp` correctly?

Reply:
✅ seems fine, I also added a FlutterError button in the example